### PR TITLE
ipehr 108 - Docs access control

### PR DESCRIPTION
**Pull request by 1db1**

### CODE REVIEW

File: src/pkg/access/list.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package access
+
+type Item struct {
+	Fields map[string][]byte
+}
+
+type List []*Item
```
Comment (SealTV) [inline src/pkg/access/list.go]: can we use here jsut `[]Item`? 
Inside Item we have only one `map` and it's is ref value

File: src/pkg/indexer/users.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+)
+
+func (i *Index) UserNew(ctx context.Context, userID string, systemID string, role uint8, pwdHash []byte, privKey *[32]byte, nonce *big.Int) (string, error) {
+	i.Lock()
+	defer i.Unlock()
+
+	var uID, sID [32]byte
+
+	copy(uID[:], []byte(userID)[:])
+	copy(sID[:], []byte(systemID)[:])
```
Comment (SealTV) [inline src/pkg/indexer/users.go]: You can copy string into array like this
```go
userID := "some_user_id_here"
	systemID := "some_system_id_here"
	var uID, sID [32]byte

	copy(uID[:], userID)
	copy(sID[:], systemID)
```

File: src/pkg/access/list.go (hunk 1 above)
Comment (1db1) [reply src/pkg/access/list.go]: yep

File: src/pkg/indexer/users.go (hunk 2 above)
Comment (1db1) [reply src/pkg/indexer/users.go]: thx

**Pull request merged**